src/main/java/com/example/rets_api/repository/PropertyRepositoryQuerydsl.java

```diff
@@ -120,6 +124,12 @@ public List<PropertyEntity> fetchAll(PropertyFilter filterParams) {
         if(filterParams.getBathroomsQty() >= 0)
             query = query.where(property.bathroomsQty.eq(filterParams.getBathroomsQty()));
 
+        if(!filterParams.getLeaseOption().equals(DEFAULT_STRING_VALUE))
+            query = query.where(financialData.leaseOption.eq(filterParams.getLeaseOption()));
+
+        if(!filterParams.getLeaseIndicator().equals(Indicator.DEFAULT_ENUM_VALUE))
+            query = query.where(financialData.leaseIndicator.eq(filterParams.getLeaseIndicator()));
+
```

Comment:
Need to add Animal policy at the filters

Reply:
added it